app/src/main/res/layout/category_edit_button.xml

```diff
@@ -53,11 +53,23 @@
         android:layout_height="@dimen/edit_categories_list_button_height"
         android:background="@drawable/button_default_background"
         android:src="@drawable/arrow_down_40dp"
+        android:layout_marginEnd="8dp"
         app:layout_constraintBottom_toBottomOf="@id/move_category_up_button"
-        app:layout_constraintEnd_toStartOf="@id/hidden_view"
+        app:layout_constraintEnd_toStartOf="@id/show_hide_category_button"
         app:layout_constraintStart_toEndOf="@id/move_category_up_button"
         app:layout_constraintTop_toTopOf="@id/move_category_up_button" />
 
+    <com.willowtree.vocable.customviews.VocableImageButton
```

Comment:
We've got three versions of `category_edit_button` so make sure they're all updated if they need to be.